.github/workflows/android-maven-release.yml

```diff
@@ -53,9 +55,31 @@ on:
           `"code/edge/src/main/java/com/adobe/marketing/mobile/EdgeConstants.java, code/gradle.properties"`
 
           This argument is required.
-          
+        type: string
         required: true
+
+      version-verify-dependencies:
+        description: >
+          A comma-separated list of dependencies with their versions. Each dependency can 
+          optionally specify the semicolon-separated list of file paths where it applies using the `@` symbol.
+          
+          **Syntax:** `<name> <version>[@file_path1[;file_path2;...]]`
+
+          - If the `@` syntax is used, the dependency will only apply to the specified files.
+          - If the `@` symbol is omitted, the dependency applies to all relevant files.
+          - When specifying custom files, you must provide either an absolute or relative path to each file.
+
+          **Example:**
+          `"AEPCore 7.8.9, AEPEdgeIdentity 8.9.10@code/gradle.properties"`
```

Comment:
Can you update the description to remove the AEP prefix? We don't need it for Android.